src/buildMathML.js

```diff
@@ -489,8 +490,7 @@ groupTypes.sizing = function(group, options) {
     // in, so we can't reset the size to normal before changing it.  Now
     // that we're passing an options parameter we should be able to fix
     // this.
-    node.setAttribute(
-        "mathsize", buildCommon.sizingMultiplier[group.value.size] + "em");
+    node.setAttribute("mathsize", newOptions.sizeMultiplier + "em");
```

Comment:
This should be checkable by rendering a expression with nested sizing commands and verify that the `mathsize` values in the MathML tree in the DOM match the sizes we use for same elements in visible HTML nodes that KaTeX renders.